src/encoded/static/data/experiment-set-matrix-demo.jsx

```diff
@@ -0,0 +1,235 @@
+    <p>This generates the following display:</p>
+    <ExperimentSetMatrix
```

Comment:
I think it would be really helpful if the actual example matrix was cleaned up a bit so extra unnecessary stuff was not included - or if it does need to be there some textual explanation as to why would be good.

I notice that for example, the search returns results for K562 cells and these are included in the matrix but K562 doesn't look like it's included in the headerColumnsOrder property.

Also you had "experiment_category": "experiments_in_set.experiment_type.experiment_type", as the first entry in the FieldChangeMap (which I changed to experiments_in_set.experiment_type.experiment_subclass_short like it is in the following example) but that doesn't seem to be used? So my change had no effect.  

While the examples are indeed very helpful it's just a bit hard to tell what is actually being used and needed to get the matrix built.

Also it looks like clicking on a box of the martrix currently is not returning the search results in the correct numbers - I don't know if this is just because it's searching data but deployed on hotseat or local?

Reply:
@aschroed Thanks for your valuable feedback. I'll add more detailed explanations for the props shortly. For first-time users, it doesn't look easy to use.

I write down a short comment below regarding your feedbacks:

> I notice that for example, the search returns results for K562 cells and these are included in the matrix but K562 doesn't look like it's included in the headerColumnsOrder property.

> Also it looks like clicking on a box of the martrix currently is not returning the search results in the correct numbers - I don't know if this is just because it's searching data but deployed on hotseat or local?

For the sake of example, queries."4DN".url explicitly starts with `"https://data.4dnucleome.org/browse/?..."` since `mastertest` returns no result in this case. In a real-world example, starting with`"/browse/?..."` is recommended. On the other hand, querying `data` from `mastertest` not returns the exact counts since we run a cross-site query. (e.g. [the query](https://data.4dnucleome.org/browse/?award.center_title=NOFIC+-+Belmont&experiments_in_set.biosample.biosource_summary=K562+%28Tier+2%29&experiments_in_set.biosample.biosource_summary=HCT116+%28Tier+2%29&experiments_in_set.biosample.biosource_summary=HFFc6+%28Tier+1%29&experiments_in_set.biosample.biosource_summary=H1-hESC+%28Tier+1%29&experiments_in_set.experiment_type.display_title=TSA-seq&experiments_in_set.experiment_type.display_title=DamID-seq&experiments_in_set.experiment_type.display_title=2-stage+Repli-seq&status=released&status=released+to+project&limit=all) returns 83 items for not logged-in, but 97 for authenticated users)